device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/MessagePropertyTest.java

```diff
@@ -46,32 +46,6 @@ public void constructorSavesPropertyValue()
 		
     }
 
-	 @Test
-    public void constructorSavesPropertyValueSlash()
-    {
-        final String name = "topic";
-        final String value = "/news/sports";
-
-        MessageProperty property = new MessageProperty(name, value);
-        String testValue = property.getValue();
-
-        final String expectedValue = value;
-        assertThat(testValue, is(expectedValue));
-    }
-    
-    @Test
-    public void constructorSavesPropertyValueTab()
-    {
```

Comment:
Same here. Tab is an illegal character, so we shouldn't have had a test like this